pkg/virtctl/vmexport/vmexport.go

```diff
@@ -456,11 +473,11 @@ func handleHTTPRequest(client kubecli.KubevirtClient, vmexport *exportv1.Virtual
 }
 
 // getHTTPClient assigns the default, non-mocked HTTP client
-func getHTTPClient(transport *http.Transport) *http.Client {
+func getHTTPClient(transport *http.Transport, insecure bool) *http.Client {
 	if insecure == true {
```

Comment:
you can discard the if

Reply:
They are mutually exclusive. I wanted to remove the rootcas when using `--insecure` for testing purposes, but I don't know if we really want users to be able to skip certificate validation.